components/bioformats

```diff
@@ -1 +1 @@
-Subproject commit b47a1fb154552609183b42d8e2f7724f99ebe3f3
+Subproject commit bf7865d3ff4679a9522819b3d480e94319fe7b7c
```

Comment:
Can you revert this submodule bump and mark a dependency in the description instead?